src/main/resources/testchipip/tlrom/rom.S

```diff
@@ -0,0 +1,9 @@
+#define DRAM_BASE 0x80000000
+
+// Dummy empty alternate boot rom.
+// Immediately jumps to DRAM_BASE
+.section .text.start, "ax", @progbits
```

Comment:
Can we just use this file as the default instead of hardcoding the array in the `SerialTLROM`? AKA compile this... commit it... and read in the file.

Reply:
Reading in the file is difficult... For the bootrom we have to copy it to a top-level directory under Chipyard with GenerateSimFiles. For FireSim we have to use a separate WithBootrom fragment with firesim specific paths.

This default tlrom is practically useless. The intention is that you would replace this with your actual boot code post-silicon. 
I added the dummy `j DRAM_BASE` for testing, to confirm that the core can jump to here, execute code, and then jump into DRAM.